content/docs/user-guide/assets/pipeline/asset-types.md

```diff
@@ -0,0 +1,26 @@
+---
+linkTitle: Asset Types 
+title: Asset Types 
+description: Asset Types supported by the Asset Pipeline Open 3D Engine (O3DE).
+weight: 700
+toc: true
+---
+
+The table below describes some of the most common source asset types supported by the **Asset Pipeline** in **Open 3D Engine (O3DE)**.
+
+| Asset file | Description  |

```

Comment:
It could be done. I think this asset list was actually started before most of the documentation and before I had a clear concept of the Asset Pipeline. We can get this going in a new PR.